src/app/feature/consultant/components/consultant-header/export-profile.ts

```diff
@@ -4,46 +4,36 @@ export class ExportProfile {
 
```

Comment:
most of these changes came from using auto formatting. the deafult i am using is 2 spaces for tabbing